ArduPlane/quadplane.h

```diff
@@ -122,6 +122,9 @@ class QuadPlane
 
     // return true if the user has set ENABLE
     bool enabled(void) const { return enable != 0; }
+    
+    // is throttle controlled landing descent active?
+    mutable bool thr_ctrl_land;
```

Comment:
instead of using mutable, better to just remove the const off QuadPlane::landing_descent_rate_cms()

Reply:
done